src/main/java/frc/robot/commands/DriveManual.java

```diff
@@ -59,8 +61,14 @@ public void execute() {
     }
 
     // Get Joystick inputs
-    double transMultiplier = slowMultiplier * redAllianceMultiplier
-        * constDrivetrain.OBSERVED_DRIVE_SPEED.in(Units.MetersPerSecond);
+    double transMultiplier = SN_Math.interpolate(
```

Comment:
The input is what you want to be converting so in this case your input would be your height.